notebooks/integrations/hugging-face/loading-model-from-hugging-face.ipynb

```diff
@@ -334,40 +197,29 @@
     "          }\n",
     "        }\n",
     "      }])"
-   ],
-   "metadata": {
-    "colab": {
-     "base_uri": "https://localhost:8080/"
-    },
-    "id": "geY7WLh7Ky-k",
-    "outputId": "97c17b36-94d6-454f-b976-4c20e8e49edc"
-   },
-   "execution_count": null,
-   "outputs": [
-    {
-     "output_type": "execute_result",
-     "data": {
-      "text/plain": [
-       "ObjectApiResponse({'acknowledged': True})"
-      ]
-     },
-     "metadata": {},
-     "execution_count": 6
-    }
    ]
   },
   {
    "cell_type": "markdown",
+   "metadata": {
+    "id": "IW-GIlH2OxB4"
+   },
    "source": [
     "### Create Index and mappings for Blogs\n",
```

Comment:
suggestion: condense the field setup to just body_content, title, url & id, as we dont use these fields for the search and simplifies for readability.